src/eradius_config.erl

```diff
@@ -228,6 +258,13 @@ validate_server(String) when is_list(String) ->
         _ ->
             {invalid, io_lib:format("bad address/port combination: ~p", [String])}
     end;
+validate_server({IP, Port, Opts}) when is_list(Opts) ->
+    case validate_server({IP, Port}) of
```

Comment:
This one possibly also should include validation of server options, like:

```
case validate_server({IP, Port}), validate_options(Opts)} of
   {{invalid, _Reason}, _} = E ->
       e;
   {_, {invalid, _Reason}} = E ->
       e;
   ....
end
```

Reply:
great suggestion! thanks

Reply:
done

Reply:
thanks